packages/mint-components/src/components/tax-and-cash/sqm-user-info-form/sqm-user-info-form.feature

```diff
@@ -0,0 +1,60 @@
+@owner:andy
+@author:andy
+
+Feature: Tax Form Step One
+
+
+	Background: A user is prompted to enter their personal and initial tax information
+		Given a user is on Tax Form Step One
+
+    @minutia
+    Scenario: The email input is pre-filled and cannot be changed
+    When the user is filling out the form
+    Then they see the email input has already been filled with their email
+    And the field is disabled
+
+
+    # AL: Need full list of countries
```

Comment:
@AndiLuo , @Locrian24 it would be worth digging into the Impact frontend UI/code and getting these figured out